swagger_server/itm/itm_action_handler.py

```diff
@@ -391,8 +404,11 @@ def tag_character(self, character: Character, tag: str):
 
     def search(self):
         """
-        Search for more characters in the scene.
+        Search for more characters.
         """
+        # After a search, the ADM is at a new location (which may or may not have patients), so all previous characters become unseen.
+        for character in self.session.state.characters:
```

Comment:
You mention that there may or may not be patients in the new location. I think I am just missing where those new potential characters are added to the state because here we are setting all of the characters to `unseen`. Because if there are patients at this new location they wouldn't be marked as unseen, right?

Reply:
Yep, the onus is on the scenario designer.  There's no new code to handle that.  Check out the `after_search` scene in the test file and you'll see that `Civilian_02` is added to the state.  And you're right-- the new character should *not* be marked unseen.